release_bp/opx_debian_base_stretch.xml

```diff
@@ -0,0 +1,30 @@
+<package_set xmlns:xi="http://www.w3.org/2001/XInclude">
+  <!-- Packages to be installed on all targets -->
+
+  <name>OpenSwitch Added Debian Stretch packages</name>
+  <type>distro</type>
+  <default_solver/>
+
+  <xi:include href="repo/stretch.xml"/>
```

Comment:
@GarrickHe @jeff-yin Still can't get bind9 from stretch-backports to install. We can go with no bind9 at all or use the regular bind9 (no backports). I opted for regular bind9 in this PR.